### ISSUE TITLE
Question on getting RIB 

### ISSUE THREAD

**Issue by ksalamat** [CAIDA/bgpstream#103]
Hi Alistair,

many thanks for the wonderful work. I am wondering if there is a simple way to do something that I think is frequently needed (if there is a way, would be great to add it into the tutorial example). Frequently, you are interested in beginning your BGP monitoring using a RIB dump and follow-up with updates. I can currently get RIB alone, updates alone, and mixed RIB and updates However, how can I get one full RIB dumps and continue with updates. One solution would be to begin with getting only RIB dumps and when all RIB dumps are received continue with only updates. But RIB dump are not aligned on time so we will have issues. Another solution will be to update in fly the bgp_filters. I am wondering if we can increment the filters after execute bgpstream_start(); ?
many thanks

**Comment by alistairking** [comment]
Hi,

To achieve this type of thing, we usually start processing from a time where all collectors dump a RIB (i.e., midnight, 8am, 4pm), and then use the RIB period filter
https://bgpstream.caida.org/docs/api/libbgpstream/bgpstream.h.html#_CPPv231bgpstream_add_rib_period_filterP11bgpstream_t8uint32_t
and set it to -1, which will give only first RIB and then no more.

Well, we actually set this RIB period filter to 86400 so that we get 1 RIB per day and then we use these RIBs to perform synchronization and sanity checking on the information we've inferred from the updates stream.

Hope this helps!
